lib/assets/components/explore_tiles.dart

```diff
@@ -333,10 +336,13 @@ class _ExploreTileTitle extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Text(
-      title,
-      style: Theme.of(context).primaryTextTheme.headline2!,
-      textScaleFactor: .6,
+    return Container(
+      width: 233,
```

Comment:
Putting the text in a container of width equal to that of the sized box in the stack widget (see below). This is to hopefully prevent going underneath the coloured part of the stack.